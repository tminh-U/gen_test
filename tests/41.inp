9
8 18 14 6 14 10 12 4 3 
8
9 9
1 7
7 7
3 4
8 9
4 4
1 6
7 7
