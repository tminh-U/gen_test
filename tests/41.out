2
21
2
3
3
2
15
2
